esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -173,8 +190,9 @@ class WsWuInfo
     * LogAccessLogFormat logFormat - Declares the log report format
     * unsigned wuLogSearchTimeBuffSecs - Defines the query time-window before wu creation and after wu end
     */
-    void readWorkunitComponentLogs(const char* outFile, unsigned maxLogRecords, LogAccessReturnColsMode retColsMode,
-                                   LogAccessLogFormat logFormat, unsigned wuLogSearchTimeBuffSecs);
+    void readWorkunitComponentLogs(const char* outFile, const unsigned maxLogRecords, const LogAccessReturnColsMode retColsMode,
```

Comment:
what is the benefit of adding const in front of the numeric parameters passed by value?